raven/1.8.1/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG RAVEN_VERSION=1.8.1
+
+# base image
+FROM ubuntu:jammy as app
+
+ARG RAVEN_VERSION
+
+# metadata
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="raven"
+LABEL software.version="${RAVEN_VERSION}"
+LABEL description="Raven is a de novo genome assembler for long uncorrected reads."
+LABEL website="https://github.com/lbcb-sci/raven"
+LABEL license="https://github.com/lbcb-sci/raven/blob/master/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# otherwise tzdata won't install
+ARG DEBIAN_FRONTEND=noninteractive
+
+# install dependencies via apt-get or yum if using a centos or fedora base 
+RUN apt-get update && apt-get install -y \
+    wget \
+    git \
+    clang \
+    gcc \
+    zlib1g-dev \
+    build-essential \
+    cmake \
+    libboost-all-dev \
+    doxygen \
+    && apt-get autoclean \
+    && rm -rf /var/lib/apt/lists/*
+
+# install raven
+RUN wget https://github.com/lbcb-sci/raven/archive/refs/tags/${RAVEN_VERSION}.tar.gz && \
+    tar -xvf ${RAVEN_VERSION}.tar.gz && \
+    rm ${RAVEN_VERSION}.tar.gz 
+  
+WORKDIR raven-${RAVEN_VERSION}
+
+RUN cmake -S ./ -B ./build -DRAVEN_BUILD_EXE=1 -DCMAKE_BUILD_TYPE=Release -Wno-dev && \
```

Comment:
I think you might be missing the intermediate step `cmake --build build` here? (disclaimer: I have not tested, just looking at the Raven docs)
```
cmake -S ./ -B./build -DRAVEN_BUILD_EXE=1 -DCMAKE_BUILD_TYPE=Release
cmake --build build
cmake --install ./build

```

Reply:
HOW'D YOU GET THAT FROM TEH RAVEN DOCS!?

Reply:
And, yes, that fixed it.

Reply:
That's what a decade of scanning bioinformatics software documentation will do for ya 😜